### PR TITLE
#159695554 Fix Codeclimate code smell

### PR DESCRIPTION
**Pull request by albert85**
#### What does this PR do?
Fix code climate code smell

#### Description of Task to be completed?
- refactor Login.jsx file
- refactor auth.js

#### How should this be manually tested?
* Clone the repository
* Cd to the project folder
* Run `chmod +x docker.sh` on the terminal
* Start the server using the following commands
    * `./docker.sh development` (for development purposes)  
    *    `./docker.sh production` (for production purposes)
* On the browser, navigate to localhost:3000 for development purposes

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159695554](https://www.pivotaltracker.com/story/show/159695554)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/redux/reducers/auth.js [hunk 1]
```diff
@@ -18,9 +18,7 @@ const auth = (state = initialState, action) => {
     };
   case types.SET_CURRENT_USER_FAILURE:
     return {
-      ...state, 
-      isAuthenticated: action.response.isAuthenticated,
```
Comment (samuelkubai) [inline src/redux/reducers/auth.js]: @albert85 where are this being assigned after you've removed them here?

Comment (albert85) [reply src/redux/reducers/auth.js]: @samuelkubai it was removed by @Spectrumsun

Comment (samuelkubai) [reply src/redux/reducers/auth.js]: @Spectrumsun please explain why you removed this.

Comment (samuelkubai) [reply src/redux/reducers/auth.js]: @Spectrumsun I have figured it out, never mind.

**Pull request merged**